lib/puppet/transport/schema/cisco_nexus.rb

```diff
@@ -0,0 +1,33 @@
+require 'puppet/resource_api'
+
+Puppet::ResourceApi.register_transport(
+  name: 'cisco_nexus',
+  desc: 'This transport connects to a Cisco Nexus Switch.',
+  connection_info: {
+    host: {
+      type: 'Optional[String]',
+      desc: 'The FQDN or IP address of the device to connect to.',
+    },
+    port: {
+      type: 'Optional[String]',
+      desc: 'The port of the device to connect to..',
+    },
+    transport: {
+      type: 'Optional[Enum["http", "https"]]',
+      desc: 'The type of transport protocol to use when connecting the the device.',
```

Comment:
`when connecting ...` has double `the`

Reply:
Maybe specify that it defaults to `http`, as this is currently the default.

Reply:
First `the` has been changed to `to` and the default has been specified.